src/components/molecules/FormFields/Credential/Credential.module.css

```diff
@@ -0,0 +1,62 @@
+.chip {
+  border: 1px solid var(--border-color);
+  display: flex;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.input {
```

Comment:
why do you duplicate all our input styles? Either use the `InputElement` component, or `compose`  the styles from the CSS module